me.njean.shootinpactris.json

```diff
@@ -0,0 +1,89 @@
+{
+    "app-id": "me.njean.shootingpactris",
+    "branch": "stable",
```

Comment:
```suggestion
```
It's already the default branch on Flathub

Reply:
Removed! I was just copy-pasting from our GIMP manifest. I guess I can remove the line from there as well then. :-)